schemas/student.py

```diff
@@ -23,49 +24,47 @@ class CreateStudentIn(BaseModel):
     gender: str
 
     @validator('full_name')
-    def validate_full_name(value):
-        full_name = value.split()
-        if not full_name:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
-        elif len(full_name) < 2:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
+    def validate_full_name(value):  # TODO may be there is need to use staticmethod decorator
+        full_name = value.split()   # TODO use hint or so
+        if not full_name or len(full_name) < 2:
+            raise BackendException(message="The student's name and surname are mandatory!")
```

Comment:
Don't use `BackendException` inside Pydantic schemas validation. it should be handled by another handler: `validation_exception_handler`.